### ISSUE TITLE
Is a Community Group the right place to develop our own Process?

### ISSUE THREAD

**Issue by dwsinger** [w3c/w3process#561]
I suspect we adopted the CG structure because it was easy, lightweight, and allowed for public visibility. But it seems strange to be using the vehicle designed to support public exploration of web-related ideas, as the vehicle to develop the Process that applies to members.

Should this be an IG with public visibility, ability to invite experts, but making it clear that the W3C are the ones with some authority, and public guests' input is appreciated and welcome but not binding?

**Comment by jeffjaffe** [comment]
Rather than addressing this question in general, I would like to comment on Process 2022 specifically.

For Process 2022 there is a great deal of momentum to revamp the process so it no longer revolves so much around the Director.

For these changes we need to build a larger community of participants (starting with our Members - but maybe not exclusively) that actively participate.  We should strive to demonstrate how the revised approach has widespread support.

**Comment by mnot** [comment]
So, if I wanted to, tomorrow I could start a CG called "Process2" and it would have just as much weight as this CG does. As a participant in this CG, it's not at all clear to me how decisions are made (in the details, and especially when there's contention), how I would object to a decision before it was taken to the AC, and so on. Despite that, the AC for some reason treats the output of this CG as legitimate, usually accepting whatever it recommends. I suspect getting the AC and especially the Team to pay attention to a fictional Process2 CG would be an uphill battle, at the least.

That seems like a _bad governance smell_ to me. If power is going to reside in this group, it should be formalised and appropriately managed. 

Developing Director-Free in an accountable, well-understood and rules-based fora is the best way to demonstrate widespread support. Non-members can and do participate in WG discussions, so that won't be a barrier.

**Comment by dwsinger** [comment]
> So, if I wanted to, tomorrow I could start a CG called "Process2" and it would have just as much weight as this CG does.

Not quite: the AB chose to delegate discussion here, but still holds the strings. Without the AB listening, or the ability to get community consensus and Director approval, another CG would be whistlin' in the wind.

> As a participant in this CG, it's not at all clear to me how decisions are made (in the details, and especially when there's contention), how I would object to a decision before it was taken to the AC, and so on. 

At the moment, we seek community agreement on multiple levels: detailed work is done in this CG, points of principle and policy are discussed in the AB, and once the AB is happy with a text, it's sent to the AC on a dual-cycle (informal review, formal vote), and W3M for comment, and finally we hope the Director approves it.

As far as I am concerned, D-F means that AC ratification is the actual last step, not an advisory one.

I'd prefer an IG as it would make clear that only members have formal influence; outside voices are welcome for comments and advice.

**Comment by michaelchampion** [comment]
> I'd prefer an IG as it would make clear that only members have formal influence; outside voices are welcome for comments and advice.

I don't have a strong opinion either way.  Mark makes some good points about it being good governance to have a formalized process to change the rules.  But maintaining the Process *is* the AB's main formal responsibility, 

it was the AB that chose to do much of the work in public, partly to get the input of people who have distanced themselves from W3C so they would tell the AB what needs to be done to win them back.  That made a lot of sense back in the days of strong disagreement with WHATWG folks about process issues such as "living standards."  Maybe that's not necessary any more?

Also, I wonder whether there is a real problem here worth investing the time (and opportunity cost) to fix.  There are only a handful of people who care enough about the formal Process to engage in detailed discussions of how to improve it..  I agree with @jeffjaffe that the more pressing challenge is to get a critical mass of people who will actively participate in generating ideas and building consensus for them.  Whether that happens in a CG, IG, WG, or simply the AB+invitees is less important than making it happen.

**Comment by plehegar** [comment]
update: the level of participation in the CG remains low. TPAC2022 was an opportunity to send a call for participation to the community (breakout, chairs, AC, etc.). No one at TPAC2022 has suggested that the CG is the wrong vehicle to make progress.

**Comment by mnot** [comment]
Not sure what you're saying ther @plehegar -- do you mean TPAC was a _lost_ opportunity, or did one go out (I didn't see it)? It's hard to give weight to what people don't suggest here - we should be doing the right thing regardless of whether someone brings it up.

**Comment by dwsinger** [comment]
I'm not aware of _complaint_, I do worry about _risk_ and being excessively public about something that is quintessentially a member matter. We lay ourselves open to being interfered with, and to 'washing our laundry in public'.

**Comment by TallTed** [comment]
If a CG is deemed inappropriate, then we need a new and special charter for, and conversion to, a long-lived and/or easily-rechartered WG, and right quick.

Also, if a main concern is to keep this "quintessential member matter" in our own laundry room, then all current and future Process work should be done in a member-only space, including the members-only WG mailing list, and extending to a restricted-access GitHub repo, etc.

**Comment by dwsinger** [comment]
I'm not sure we need a WG (they are usually used to develop rec-track specs with patent commitments); an IG may suffice. I'm not saying that everything _has_ to be member-only, just that in a CG is _is_ all public (there's no discretion), and public and members of the consortium are roughly on an equal footing.

**Comment by dwsinger** [comment]
Nor do I think this is urgent...

**Comment by jeffjaffe** [comment]
In general, W3C's preference has been to work in the public.

Specifically, for our process evolution, we have operated in public for many years.  The results have been (imho) quite successful.  We have enhanced our process on an almost annual basis.  We have introduced huge change: dropping Last Call, adding CEPC, living standards, registries.  We have good momentum on Director-free.

My preference would be to continue what has worked well, unless there has been some specific problem identified.

**Comment by dwsinger** [comment]
I'm having a hard time thinking of a case where equal-basis public involvement and exposure was actively helpful. Can anyone?

**Comment by TallTed** [comment]
(Because others may have the same mental hiccup, `IG` above now clearly was meant to mean [`Interest Group`](https://www.w3.org/groups/ig/), as opposed to the long-ago `Incubation Group` which was mostly subsumed into today's concept of a [`Community Group`](https://www.w3.org/groups/cg/).)

**Comment by fantasai** [comment]
I agree with @dwsinger that this should be chartered into an Interest Group rather than a CG, and also that it is not urgent to do so.

We don't need to make it a Member-restricted IG, it can be a publicly-visible IG that continues to use this existing repository, and I think that's fine. (In fact, unless we fix the problem that most Invited Experts do not have Member access, I would formally object to making it a Member-only IG.) But I think it makes sense to be formally chartered and to draw official participants from the membership of W3C rather than the public at large.

**Comment by mnot** [comment]
That seems reasonable, and I agree that it's not urgent. Depending on the timelines, it might be nice to get in before director-free ships, since that's the biggest change on the horizon.

**Comment by mnot** [comment]
Now that we're (hopefully) shipping a new Process, perhaps it's time to reconsider this?

**Comment by frivoal** [comment]
I agree that a CG is a little peculiar (though lightweight and convenient), but this isn't really acting like an ordinary CG, since it is acting under delegation from the AB. I'm not against a little more formalism, but I am unsure that an IG is the right approach: it would be atypical for a IG to have a charter making it subordinate to the AB, which is how this group operates. A taskforce of the AB, open to broad participation, would seem more appropriate to me.

Whether IG or CG or Taskforce AB, I do think having a clear charter wouldn't hurt. Then it'd be clear why this group (and not some other) is empowered to do its work on the Process, and how it operates.

**Comment by mnot** [comment]
+1 to that -- the exact form is unimportant; it just needs to be properly constituted.

**Comment by dwsinger** [comment]
I also think it peculiar that we use a structure open to the public. A TF of the AB, open to members, would seem much better.

**Comment by fantasai** [comment]
Since this topic is coming up again, I've [drafted a possible charter for a Process Interest Group](https://lists.w3.org/Archives/Public/public-w3process/2025Jan/att-0001/process-ig-draft.html) that attempts to capture the work mode of this CG.

**Comment by plehegar** [comment]
I think the draft is a good start for discussions.

Is the AB having an overarching discussion on how it organizes its work items (Vision, Process, Patent Policy, Code of Conduct, etc.) ?

If the answer is yes, then shouldn't we align how those work items get done/discussed ?

I did a quick comparison with PSIG:
  https://www.w3.org/2025/01/psig-charter.html

In my mind, both groups act pretty much the same way:
- they propose or suggest changes to the AB for their respective documents (Process, Patent Policy) but don't have a formal role.
- they both are empowered to create adjunct documentation to help implement those documents. PSIG produce a FAQ and is responsible for it. Process CG did not produce documentation so far but should feel free to do so (those produced by the AB have been incorporated in the Guidebook overtime, such as [1]).
- they both should feel free to review/comment on adjunct documentation/tools produced by the Team to implement those documents (IPR pages/CfE emails/etc, Transition requests/AC email notices/etc).

With that in mind, I do believe that the group should, to some level, care about the Guidebook, since it's implementing the Process. The Team is responsible for the Guidebook but we need a forum where discussion around the Guidebook can happen. If the Guidebook was inconsistent with the Process, the group (or the AB) would be in its rights to discuss it. I already used the CG in the past to discuss some changes in the Guidebook.

One suggestion: remove Guidebook from out of scope and add in scope (or something equivalent):
[[
The Group may suggest changes to W3C Guidebook, but is not responsible for formally documenting or implementing it.
]]

**Comment by fantasai** [comment]
PSIG will probably remain somewhat exceptional, but in general I agree.

**Comment by nigelmegitt** [comment]
>Process CG did not produce documentation so far 

@plehegar is the Process not documentation?

**Comment by plehegar** [comment]
> > Process CG did not produce documentation so far
> 
> [@plehegar](https://github.com/plehegar) is the Process not documentation?

I should said "adjunct documentation"

**Comment by fantasai** [comment]
Updated the [draft of a possible Process Interest Group](https://lists.w3.org/Archives/Public/public-w3process/2025Apr/0001.html) to [incorporate plh's comments](https://github.com/w3c/process/issues/561#issuecomment-2590399146).
